### DISCUSSION TITLE
add a position number on the waiting list

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [pretix/pretix#2546] [Ideas]
Would be great if the waiting lists would add a number column to quickly see at which "spot" an entry on the waiting list is

**Comment by raphaelm** [comment]
Hmm, not sure if that makes sense, as soon as multiple products with interlinked quotas are in use, there's no trivial way to come up with that number.

**Reply by FlohGro-dev** [reply]
Really? Shouldn't this be possible based on the defined quotas?

**Reply by raphaelm** [reply]
Let's say we have Product A and Product B.
Then, we have quota A mapping to product A, quota B mapping to product B, and quota C mapping to *both products*.

Now the first entry on the waiting list is for product A, and the second one for product B. 

Depending on which quotas are sold out and become available again in what order, it is not clear which entry gets notified first.

What numbers would we display in the backend? 1+2? Or 1+1? What information would either of these numbers provide?

**Comment by FlohGro-dev** [comment]
ok - didn't think of combined quotas.

It would be helpful to see the numbers in simpler scenarios

we have many products in a shop which all are mapped to just one quota so it would be easy to determine the number, but I understand your concerns :)
